src/sentry/sentry_apps/api/endpoints/installation_details.py

```diff
@@ -39,26 +43,35 @@ def get(self, request: Request, installation) -> Response:
         )
 
     def delete(self, request: Request, installation) -> Response:
-        installation = SentryAppInstallation.objects.get(id=installation.id)
+        sentry_app_installation: SentryAppInstallation = SentryAppInstallation.objects.get(
```

Comment:
You shouldn't need to typehint this. Was mypy not figuring out the type?

Reply:
ah, we can get rid of this, the typehinter was going between Any & SentryAppInstallation before, but renaming fixes that

Reply:
yeah mypy can't deal with a variable changing types. Having a separate variable name for the string parameter and object will help.